tvb_nest/simulator_tvb/simulator.py

```diff
@@ -67,23 +64,21 @@ class Simulator(SimulatorTVB):
     boundary_fun = None
     simulate_nest = None
 
-    model = models.Model(
+    model = Attr(
+        field_type=models.Model,
         label="Local dynamic model",
-        default=ReducedWongWangExcIOInhI,
+        default=ReducedWongWangExcIOInhI(),
         required=True,
-        order=5,
         doc="""A tvb.simulator.Model object which describes the local dynamic
             equations, their parameters, and, to some extent, where connectivity
             (local and long-range) enters and which state-variables the Monitors
             monitor. By default the 'ReducedWongWang' model is used. Read the 
             Scientific documentation to learn more about this model.""")
 
-    monitors = monitors_m.Monitor(
+    monitors = List(
+        of=monitors.Monitor,
         label="Monitor(s)",
-        default=monitors_m.Raw,
-        required=True,
```

Comment:
required=True has been missed here

Reply:
Required=True has been removed because List from _attr.py (that defines monitors) hasn't the attribute 'required' as parameter. Yet, because of the way this class is written, the attribute required=True is set internally in List.

Reply:
Thank you for the explanation. Then your change makes sense.